### PR TITLE
Fix label overflow @ config-node palette

### PR DESCRIPTION
**Pull request by ralphwetzel**
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes
This is another attempt to fix the issue raised in #3572.

As-Is (3.0.0-beta.4):
image

To-Be:
image

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: packages/node_modules/@node-red/editor-client/src/sass/tab-config.scss [hunk 1]
```diff
@@ -54,6 +54,9 @@ ul.red-ui-sidebar-node-config-list {
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        &:not(:last-child) {
```
Comment (bonanitech) [inline packages/node_modules/@node-red/editor-client/src/sass/tab-config.scss:57]: @ralphwetzel, out of curiosity, why did you add `&:not(:last-child)`?

Comment (ralphwetzel) [reply packages/node_modules/@node-red/editor-client/src/sass/tab-config.scss:57]: This adaptation of width is only necessary if the label shares its parent space with another element.
Do you think, this constraint is not necessary?

Comment (bonanitech) [reply packages/node_modules/@node-red/editor-client/src/sass/tab-config.scss:57]: I don't think so. Unless I'm missing something
